.gitignore

```diff
@@ -13,3 +13,4 @@ web/app/yarn-error.log
 **/*.gogen*
 **/*.swp
 charts/*/charts/
+charts/*/add-ons/*/charts
```

Comment:
I think we can replace these two lines with
```
charts/**/charts/*
```

Reply:
Done.